### PR TITLE
feat:freshteam-iparams-json

### PR DESCRIPTION
**Pull request by ajithajay**
## Proposed changes

Added sample app to demonstrate the Iparams JSON for Freshteam

## Types of changes

- New feature - iparams-json sample app for Freshteam

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/freshdesk/marketplace-sample-apps/blob/master/CONTRIBUTING.MD) doc
- [x] I run the app locally after the change. It works as expected.
- [x] I run the app after publishing it as a custom app after the change. It works as expected.

### CODE REVIEW

**Comment by ajithajay** [comment]
Hi @teja-kummarikuntla @zachjonesnoel  , Did you get a chance to review this PR

**Comment by ajithajay** [comment]
Hi @teja-kummarikuntla @zachjonesnoel , Did you get a chance to review this PR

**Comment by ajithajay** [comment]
Hi @teja-kummarikuntla @zachjonesnoel  , Did you get a chance to review this PR

**Comment by ajithajay** [comment]
Hi @zachjonesnoel, Did you get a chance to review this PR

File: freshteam/iparams-json/app/app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+                client.data.get('jobPosting')
+                    .then(function(data) {
+                        $('#apptext').text("Job Title: " + data.title);
+                    })
+                    .catch(function(e) {
+                        console.log('Exception - ', e);
+                    });
+        });
+    });
+});
```
Comment (zachjonesnoel) [inline freshteam/iparams-json/app/app.js]: App looks great, can you add in `client.iparams.get()` retrieving iparam values and displaying them on the app?

**Review by zachjonesnoel — CHANGES_REQUESTED** [review]
Can you showcase the feature of retrieving iparam values in the app? The iparams to demonstrate different fields is great! 👍

File: freshteam/iparams-json/app/app.js (hunk 1 above)
Comment (ajithajay) [reply freshteam/iparams-json/app/app.js]: Sure, if we include client.iparams.get() it gives more clarity. we will let you know once changes have been done

Comment (ajithajay) [reply freshteam/iparams-json/app/app.js]: @zachjonesnoel Included feature of retrieving iparams values in the app. Please review & let us know the update

Comment (ajithajay) [reply freshteam/iparams-json/app/app.js]: @zachjonesnoel Did you get a chance to review the changes?

**Comment by ajithajay** [comment]
Hi @zachjonesnoel, did you get a chance to review the changes

**Comment by ajithajay** [comment]
Hi @zachjonesnoel @teja-kummarikuntla did you get a chance to review the changes?

**Comment by ajithajay** [comment]
Hi @zachjonesnoel @teja-kummarikuntla did you get a chance to review the changes?

**Comment by ajithajay** [comment]
Hi @zachjonesnoel @teja-kummarikuntla did you get a chance to review the changes?

**Review by zachjonesnoel — APPROVED** [review]
Looks good! 👍

**Pull request merged**